packages/cli/src/utils.ts

```diff
@@ -305,3 +323,7 @@ export function isSubdir(parent: string, dir: string): boolean {
   const relative = path.relative(parent, dir);
   return !!relative && !/^..($|\/)/.test(relative) && !path.isAbsolute(relative);
 }
+
+export function relativeFilePath(filePath: string): string {
+  return filePath.replace(process.cwd(), '');
```

Comment:
```suggestion
  return path.relative(process.cwd(), filePath);
```

But maybe we don't need a separate function for it.

Reply:
yeah, thx, we can remove it I guess.